### ISSUE TITLE
Check for `CHANGELOG.md` entry for new version prior to release in `cargo release`

### ISSUE THREAD

**Issue by alilleybrinker** [mitre/hipcheck#138]
To make sure we don't forget to update the `CHANGELOG.md` for a new version before a release, we should configure `cargo-release` to check first and refuse to proceed if it's missing.

**Comment by j-lanson** [comment]
@alilleybrinker is this still a relevant issue? We don't use `cargo release` anymore.

**Comment by alilleybrinker** [comment]
Nope, not relevant anymore since we stopped maintaining a changelog file and now do updates through the project blog. Thanks!
